lib/kennel/parts_serializer.rb

```diff
@@ -48,13 +49,7 @@ def directories_to_clean_up
     end
 
     def old_paths
-      directories_to_clean_up.flat_map do |path|
-        if File.exist?(path)
-          Find.find(path).to_a
-        else
-          []
-        end
-      end.to_set
+      Dir["{#{directories_to_clean_up.join(",")}}/**/*"].to_set
```

Comment:
Edge case that this implementation introduces, FYI: PROJECT=foo, where a project with kennel_id=foo exists, but has no parts. Pre-PR: `generated/foo` would be deleted. Post-PR, it wouldn't.